packages/fxa-content-server/server/lib/routes/get-metrics-flow.js

```diff
@@ -136,6 +141,25 @@ module.exports = function(config) {
         metricsData,
         req
       );
+    } else if (metricsData.form_type === FORM_TYPE_SUBSCRIBE) {
```

Comment:
3 lookups against the same variable is probably the point at which I'd change this block to a `switch` or, even better, just have a map of `form_type` to `type` properties so that we can ditch the condition entirely and emit the events declaratively (if you see what I mean).